README.md

```diff
@@ -41,19 +41,49 @@ products, client work, and open source. [More info here](http://www.shakacode.co
 ## How is different than the [react-rails gem](https://github.com/reactjs/react-rails)?
 1. `react_on_rails` depends on [webpack](http://webpack.github.io/). `react-rails` integrates closely with sprockets and
     helps you integrate JSX and the react code into a Rails project.
-2. Likewise, using Webpack as shown in the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/)
+2. Likewise, using webpack as shown in the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/)
    does involve some extra setup. However, we feel that tight and simple integration with the node ecosystem is more than
    worth any minor setup costs.
 3. `react-rails` depends on `jquery-ujs` for client side rendering. `react_on_rails` has it's own JS code that does not
    depend on jquery.
 
 ## Installation Checklist
-1. Include the gems `react_on_rails` and `therubyracer` like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/361f4338ebb39a5d3934b00cb6d6fcf494773000/Gemfile#L42) and run `bundle`. Note, you can sustitute your preferable JavaScript engine.
+1. Include the gems `react_on_rails` and `therubyracer` like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/361f4338ebb39a5d3934b00cb6d6fcf494773000/Gemfile#L42) and run `bundle`. Note, you can substitute your preferable JavaScript engine.
   
   ```ruby
   gem "react_on_rails"
   gem "therubyracer"
   ```
+
+### Installing With The Generators
+1. The install generator will get you started with the necessary directories and configurations.
+   ```
+   rails generate react_on_rails:install
+   ```
+
+   This will create a client directory in your application root path that contains directories for your React files and webpack, NPM, and linting configurations.
+
+   The config file will be copied to `config/initializers/react_on_rails.rb`.
+
+   Foreman can be run with `foreman start -f Procfile.dev`
+
+#### Optional Generator Parameters
+1. Server rendering:
+   ```
+   rails generate react_on_rails:install --with-server-rendering
+   ```
+2. Redux support:
+   ```
```

Comment:
formatting is not correct

Please check how this looks on github

I think you need:

blank line above the ```

and use ```bash for shell stuff